src/views/Rules/ListRules/index.jsx

```diff
@@ -249,9 +297,35 @@ function ListRules(props) {
       throw error;
     }
 
-    setRulesWithScheduledChanges(
-      rulesWithScheduledChanges.filter(i => i.rule_id !== dialogRule.rule_id)
-    );
+    if (Object.keys(dialogRule.requiredSignoffs).length > 0) {
+      // A change was scheduled, we need to update the card
+      // to reflect that.
+      try {
+        const sc = (await getScheduledChange(dialogRule.rule_id)).data
+          .scheduled_changes[0];
+
+        setRulesWithScheduledChanges(
+          rulesWithScheduledChanges.map(i => {
+            if (i.rule_id !== sc.rule_id) {
+              return i;
+            }
+
+            i.scheduledChange = sc;
+
+            return i;
+          })
+        );
+      } catch (e) {
+        throw new Error(
+          'Scheduled change created but page failed to properly update page. Please refresh.'
+        );
```

Comment:
If the schedule change was created, is there a reason we would receive an error?

Reply:
It's quite unlikely, but if, eg: Balrog went down, or the GET method that we call broke after a deploy, we could get an error.

Reply:
If it breaks or Balrog went down, then there are no guarantees that the schedule change was created right?

Reply:
I think if we get to this point it has been created (because `delRule` would've thrown otherwise). In any case, I removed this message locally, and just let the onError handler show the original exception.